silk/model_factory.py

```diff
@@ -226,6 +226,16 @@ def construct_request_model(self):
         query_params = self.query_params()
         path = self.request.path
         view_name = self.view_name()
+        content_type = self.request.content_type
+
+        if content_type.startswith('multipart/form-data'):
+            body = {
+                'form_data': self.request.POST.dict(),
+                'files': {f: self.request.FILES[f].name for f in self.request.FILES}
+            }
+            raw_body = None  # Raw body is not available for multipart/form-data
+        else:
+            body, raw_body = self.body()
```

Comment:
@ziyacivan Good catch, and thank you for making this patch! 👏🏼

Reply:
Oh thank you so much! I couldn't continue to the development process (means is tests) because of my social life issues, thank you again 👏🏽